app/assets/javascripts/admin/addon/services/admin-sidebar-state-manager.js

```diff
@@ -1,18 +1,28 @@
 import { tracked } from "@glimmer/tracking";
 import Service, { service } from "@ember/service";
-import { TrackedObject } from "@ember-compat/tracked-built-ins";
 import KeyValueStore from "discourse/lib/key-value-store";
 import { ADMIN_PANEL } from "discourse/lib/sidebar/panels";
 
 export default class AdminSidebarStateManager extends Service {
   @service sidebarState;
   @service currentUser;
-  @tracked keywords = new TrackedObject();
+  @tracked keywords = {};
```

Comment:
`this.keywords` isn't reassigned at any point so `@tracked` isn't needed:

```suggestion
  keywords = {};
```

Reply:
Ah yeah dumb oversight, thanks!